pkgs/development/python-modules/mayavi/default.nix

```diff
@@ -26,6 +27,24 @@ buildPythonPackage rec {
     hash = "sha256-n0J+8spska542S02ibpr7KJMhGDicG2KHJuEKJrT/Z4=";
   };
 
+  patches = [
```

Comment:
I think `python311Packages.mayavi` is broken on master as well?

Reply:
Probably due to the lack of Python 3.11 compatibility, but the incompatible function pointer conversion error would be specific to staging-next due to the clang 16 upgrade.

Should I split the commit or separate it out and target master?

Reply:
> Probably due to the lack of Python 3.11 compatibility, but the incompatible function pointer conversion error would be specific to staging-next due to the clang 16 upgrade.

Oh, then please keep it in this PR.